docs/get-started/deploy-cloudfoundry-for-enterprise.md

```diff
@@ -0,0 +1,129 @@
+# Deploy Cloud Foundry using multiple storage accounts and availability sets
+
+This document provides additional configuration options for high availability and performance of your Cloud Foundry infrastructure.
+For basic Cloud Foundry deployment steps, click [**HERE**](./deploy-cloudfoundry.md).
+ 
+## Availability Set
+
+### Why
+
+For high availability of your Cloud Foundry infrastructure, you can utilize Availability Set for each of your resource pools.
+
+There are two types of Microsoft Azure platform events that can affect the availability of your virtual machines: planned maintenance and unplanned maintenance. To reduce the impact of downtime due to one or more of these events, we recommend configuring multiple virtual machines in an [Availability Set](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-manage-availability/) for redundancy.
+
+### How
+
+1. Open your manifest, and configure an availability set into your `resource_pools`.
+
+  ```
+  - name: resource_router
+    network: default
+    stemcell:
+      name: bosh-azure-hyperv-ubuntu-trusty-go_agent
+      version: '0000'
+    cloud_properties:
+      availability_set: <availability-set-name>
+      instance_type: Standard_D1
+  ```
+
+  If `<availability-set-name>` does not exist, it will be created automatically.
+
+2. Specify the resource_pool as `resource_router` in your job. Then these 2 VMs will be put into the availability set `<availability-set-name>`.
+
+  ```
+  - name: router_z1
+    instances: 2
+    resource_pool: resource_router
+    templates:
+    - {name: gorouter, release: cf}
+    - {name: metron_agent, release: cf}
+    networks:
+    - name: default
+      static_ips: [10.0.16.12, 10.0.16.22]
+    properties:
+      dropsonde: {enabled: true}
+  ```
+
+3. Deploy your Cloud Foundry.
+
+## Multiple Storage Account
+
+### Why
+
+Due to [Azure Storage Limits](https://azure.microsoft.com/en-us/documentation/articles/azure-subscription-service-limits/#storage-limits), the maximum number of highly utilized disks in one storage account is about 40 (20,000/500 IOPS per disk) for a Standard Tier VM. If you need high-performance, low-latency disk support, you can use [Azure Premium Storage](https://azure.microsoft.com/en-us/documentation/articles/storage-premium-storage-preview-portal/). For premium storage disks (P30), the maximum number of highly utilized disks in one storage account is about 31.
+
+If the number of disks in one storage account exceeds the maximum number, you need to split these disks into multiple storage accounts for performance optimizations.
+
+### How
+
```

Comment:
Suggestion for "How" section:
1. Decide whether you need multiple storage accounts and how many of them
Since one storage account can hold up to 40 disks, if your deployment needs less than 40 disks, no need for multiple storage accounts. If you need more than 40 disks, then you need one additional storage account for every additional 40 disks. Please note, by default, one disk is created for a single VM, however you can also increase the number of disks per VM based on your need.  For example, if your deployment has 80 runners (DEAs), and assuming you need 1 disk for each VM,  then the total number of disks is 80. Therefore you need 2 storage accounts, and hence 2 resource pools for runners.